keyboards/converter/ps2_usb/rules.mk

```diff
@@ -0,0 +1,23 @@
+MCU = atmega32u4
+F_CPU ?= 16000000
+ARCH ?= AVR8
+F_USB ?= $(F_CPU)
+OPT_DEFS += -DSUSPEND_MODE_STANDBY
+BOOTLOADER_SIZE = 4096
```

Comment:
If you know the bootloader being used, it should be set directly. (See the sample below.)

If not, then this is fine. (Resolve conversation.)

https://github.com/qmk/qmk_firmware/blob/d0fb7005d51be7c876e63e87778d080c8a733a13/quantum/template/avr/rules.mk#L42-L59

Reply:
I have reflashed some of my Pro Micro boards to LUFA DFU bootloader, but some of them have default Caterina bootloader. The have the same size (4096 bytes), but you are right: `BOOTLOADER = caterina` by default seems to be more logical.

Reply:
Yup, and you can add `BOOTLOADER = lufa-dfu` to the keymaps to override this behavior, when needed. (it's what I do, actually)

Reply:
(for ease):
```suggestion
BOOTLOADER = caterina
```